.github/workflows/ci.yml

```diff
@@ -1013,6 +1013,31 @@ jobs:
           user_name: github-actions[bot]
           user_email: github-actions[bot]@users.noreply.github.com
 
+  release-app-store:
```

Comment:
Regarding App Store Connect auto sending to review: it seems like there's no such actions to upload and send an application to review, yet the REST API does indeed has a query for doing so: https://developer.apple.com/documentation/appstoreconnectapi/create_a_review_submission

But, IMHO, we should probably set up a proper meta-information modification process. We should store the screenshots to upload to App Store somewhere in the repository. Then we might even use those screenshots in the README.md of the repository. Also the application's description may be placed somewhere in the repository, so it's accessible and easy to modify with each prepare new version PR.

If we'd choose to do it manually via REST queries, this might require some work to accomplish. However, there's also such utility as [fastlane](https://fastlane.tools), which has the ability to do everything we should have: from uploading the screenshots and updating the description up to sending the app for a review.

Reply:
Discussed: we should separate this work into two PRs:
1) In this one we deploy only the beta versions (`internal` track and TestFlight).
2) In the following one we bootstrap the fastlane with all the metadata deployment.